spec/features/card_spec.rb

```diff
@@ -0,0 +1,14 @@
+# encoding: utf-8
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.